index.html

```diff
@@ -293,7 +293,7 @@ <h2 class="section-header">Connect with us</h2>
           <a href="#">Home</a> <span> | </span>
           <a href="#">Privacy Policy</a> <span> | </span>
           <a href="#">Sitemap</a> <span> | </span><br />
-          <a href="#">100 M St SE #600 Washington, DC 20003</a>
+          <a class="address" href="#">100 M St SE #600 Washington, DC 20003</a>
```

Comment:
Instead of `class="address"`, you can use the `<address></address>` native HTML5 element. This is considered to be more semantic.

Reply:
Corrected 👍